src/Logging/Logging.EventSource/src/Microsoft.Extensions.Logging.EventSource.csproj

```diff
@@ -6,13 +6,17 @@
     <PackageTags>$(PackageTags);EventSource;ETW</PackageTags>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <IsShipping>true</IsShipping>
+    <NoWarn>$(NoWarn);CS3021</NoWarn>
   </PropertyGroup>
 
   <ItemGroup>
     <Compile Include="../../shared/*.cs" />
 
+    <Reference Include="Microsoft.Bcl.Json.Sources">
```

Comment:
Does this library have to support netstandard2.0?

Reply:
Yes, it was `netstandard2.0` in `2.2` and I don't think we have plans to change that. Most of `Microsoft.Extensions.*` is designed to remain `netstandard2.0`